app/scenes/Login/Notices.tsx

```diff
@@ -21,12 +21,6 @@ export default function Notices() {
           installation. Try another?
         </NoticeAlert>
       )}
-      {notice === "hd-not-allowed" && (
-        <NoticeAlert>
-          Sorry, your Google apps domain is not allowed. Please try again with
-          an allowed team domain.
-        </NoticeAlert>
-      )}
```

Comment:
Copied the error message from this older error

Also realized this error was dead code now, so cleaned it up!